SimpleToast is blocking tvOS preview

For a multi platform projects, we can conditional link it to exclude tvOS in building and also use #canImport in Swift code.

<img width="438" alt="image" src="https://github.com/sanzaru/SimpleToast/assets/43724855/fe2ab430-5b05-4d72-a28d-d5f458c76399">

This works fine for building and archiving. However this will cause an issue when previewing SwiftUI view on tvOS.

```
== PREVIEW UPDATE ERROR:

    SchemeBuildError: Failed to build the scheme ”Forumate”
    
    'DragGesture' is unavailable in tvOS
    
    Compile SimpleToast.swift (arm64):
    /Users/kyle/Library/Developer/Xcode/DerivedData/Forumate-cfcvrozqvmyfrfcrzfasktaukvys/SourcePackages/checkouts/SimpleToast/Sources/SimpleToast/SimpleToast.swift:30:9: error: 'DragGesture' is unavailable in tvOS
            DragGesture()
            ^~~~~~~~~~~
```

--- a/Package.swift
+++ b/Package.swift
@@ -1,10 +1,10 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.7
 import PackageDescription
 
 let package = Package(
     name: "SimpleToast",
     platforms: [
-        .iOS(.v13), .macOS(.v10_15)
+        .iOS(.v13), .macOS(.v10_15), .tvOS(.v16)
     ],
     products: [
         .library(

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@
 [![](https://tinyurl.com/3e9pprjv)](https://swiftpackageindex.com/sanzaru/SimpleToast)
 [![](https://tinyurl.com/mtkte8as)](https://swiftpackageindex.com/sanzaru/SimpleToast)
 
-SimpleToast is a simple, lightweight, flexible and easy to use library to show toasts / popup notifications inside iOS or MacOS applications in SwiftUI. Because of the flexibility to show any content it is also possible to use the library for showing simple modals.
+SimpleToast is a simple, lightweight, flexible and easy to use library to show toasts / popup notifications inside iOS or macOS applications in SwiftUI. Because of the flexibility to show any content it is also possible to use the library for showing simple modals.
 
 You decide the content, the library takes care about the rest.
 

--- a/Sources/SimpleToast/SimpleToast.swift
+++ b/Sources/SimpleToast/SimpleToast.swift
@@ -25,6 +25,7 @@ struct SimpleToast<SimpleToastContent: View>: ViewModifier {
 
     private let maxDelta: CGFloat = 20
 
+    #if !os(tvOS)
     /// Dimiss the toast on drag
     private var dragGesture: some Gesture {
         DragGesture()
@@ -71,6 +72,7 @@ struct SimpleToast<SimpleToastContent: View>: ViewModifier {
                 offset = .zero
             }
     }
+    #endif
 
     @ViewBuilder
     private var toastRenderContent: some View {
@@ -80,13 +82,16 @@ struct SimpleToast<SimpleToastContent: View>: ViewModifier {
                 case .slide:
                     toastInnerContent
                         .modifier(SimpleToastSlide(showToast: $showToast, options: options))
+                        #if !os(tvOS)
                         .gesture(dragGesture)
+                        #endif
 
                 case .scale:
                     toastInnerContent
                         .modifier(SimpleToastScale(showToast: $showToast, options: options))
+                        #if !os(tvOS)
                         .gesture(dragGesture)
-
+                        #endif
                 case .skew:
                     toastInnerContent
                         .modifier(SimpleToastSkew(showToast: $showToast, options: options))
@@ -100,7 +105,9 @@ struct SimpleToast<SimpleToastContent: View>: ViewModifier {
                 default:
                     toastInnerContent
                         .modifier(SimpleToastFade(showToast: $showToast, options: options))
+                        #if !os(tvOS)
                         .gesture(dragGesture)
+                        #endif
                 }
             }
             .onTapGesture(perform: dismissOnTap)

